mage_ai/api/presenters/PipelineRunPresenter.py

```diff
@@ -45,18 +44,24 @@ async def present(self, **kwargs):
 
             return self.model.to_dict(include_attributes=additional_attributes)
         elif constants.DETAIL == kwargs['format']:
-            block_runs = self.model.block_runs
             data = self.model.to_dict()
-            pipeline_schedule = self.resource.pipeline_schedule
 
-            arr = []
-            for r in block_runs:
-                block_run = r.to_dict()
-                block_run['pipeline_schedule_id'] = pipeline_schedule.id
-                block_run['pipeline_schedule_name'] = pipeline_schedule.name
-                arr.append(block_run)
-            arr.sort(key=lambda b: b.get('created_at'))
-            data['block_runs'] = arr
+            exclude_block_runs = query.get('exclude_block_runs', [False])
```

Comment:
maybe we can do it like this to keep things consistent between different APIs: https://github.com/mage-ai/mage-ai/blob/master/mage_ai/api/presenters/GitCustomBranchPresenter.py#L33

Reply:
ok